module_3/src/com/epam/stringasstringorstringbuilder/StringModifier.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.stringasstringorstringbuilder;
+
+public class StringModifier {
+
+    // task_02
+    public static String insertTextAfterSpecifiedSubstring(String str, String textToInsert, String specifiedSubstr) {
+        String resultString = "";
+        if (str.contains(specifiedSubstr)) {
+            resultString = str.replace(specifiedSubstr, specifiedSubstr + textToInsert);
+        }
+        return resultString;
```

Comment:
I also think that you don't need to check if there were any replacement, you can just return:
```suggestion
        return str.replace(specifiedSubstr, specifiedSubstr + textToInsert);
```
Code becomes more beautiful:)